corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -296,19 +292,52 @@ FormplayerFrontend.module("Menus.Views", function (Views, FormplayerFrontend, Ba
         },
     });
 
+    Views.buildCaseTileStyles = function (tiles, numRows, numColumns, numEntitiesPerRow, useUniformUnits, prefix) {
+        var cellLayoutStyle = buildCellLayout(tiles, prefix);
+        var cellGridStyle = buildCellGridStyle(numRows,
+            numColumns,
+            numEntitiesPerRow,
+            useUniformUnits,
+            prefix);
+        if (numEntitiesPerRow > 1) {
+            var cellContainerStyle = buildCellContainerStyle(numRows, numColumns, numEntitiesPerRow);
+            return [cellLayoutStyle, cellGridStyle, cellContainerStyle];
+        } else {
+            return [cellLayoutStyle, cellGridStyle];
+        }
+    };
+
     Views.CaseTileListView = Views.CaseListView.extend({
         childView: Views.CaseTileView,
         initialize: function (options) {
             Views.CaseTileListView.__super__.initialize.apply(this, arguments);
+
+            var numEntitiesPerRow = options.numEntitiesPerRow || 1;
+            var numRows = options.maxHeight;
+            var numColumns = options.maxWidth;
+            var useUniformUnits = options.useUniformUnits;
+
+            var caseTileStyles = Views.buildCaseTileStyles(options.tiles, numRows, numColumns,
+                numEntitiesPerRow, useUniformUnits, 'list');
+
             var gridPolyfillPath = FormplayerFrontend.request('gridPolyfillPath');
-            generateCaseTileStyles(options.tiles);
-            makeInnerGridStyle(options.maxHeight,
-                options.maxWidth,
-                options.numEntitiesPerRow || 1,
-                options.useUniformUnits);
+
+            $("#list-cell-layout-style").html(caseTileStyles[0]).data("css-polyfilled", false);
+            $("#list-cell-grid-style").html(caseTileStyles[1]).data("css-polyfilled", false);
+            // If we have multiple cases per line, need to generate the outer grid style as well
+            if (caseTileStyles.length > 2) {
+                $("#list-cell-container-style").html(caseTileStyles[2]).data("css-polyfilled", false);
+            }
+
             $.getScript(gridPolyfillPath);
         },
 
+        childViewOptions: function () {
+            var dict = Views.CaseTileListView.__super__.childViewOptions.apply(this, arguments);
+            dict['prefix'] = 'list';
```

Comment:
you can also add `prefix` as a property on the view itself. that way you can just do `this.prefix`